### PR TITLE
[DOCS] CubeMap

### PR DESCRIPTION
**Pull request by NathanWarden**

### CODE REVIEW

File: doc/classes/CubeMap.xml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CubeMap" inherits="Resource" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		A CubeMap is a 6 sided 3D texture.
 	</brief_description>
 	<description>
+		A CubeMap is a 6 sided 3D texture typically used for faking reflections. This usually delivers the best performace to make an object look as if it is reflecting it's surroundings.
```
Comment (Noshyaar) [inline doc/classes/CubeMap.xml]: performance, its

File: doc/classes/CubeMap.xml [hunk 2]
```diff
@@ -19,6 +21,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Return the height of the CubeMap.
```
Comment (cbscribe) [inline doc/classes/CubeMap.xml]: - Return -> Returns
- Per the writing guidelines, use possessive:

Returns the CubeMap's height.

File: doc/classes/CubeMap.xml (hunk 1 above)
Comment (NathanWarden) [reply doc/classes/CubeMap.xml]: Thanks, it was a little difficult to read. I changed it to "[...] performance, it makes [...]".

File: doc/classes/CubeMap.xml [hunk 3]
```diff
@@ -33,6 +36,7 @@
 			<argument index="0" name="side" type="int" enum="CubeMap.Side">
 			</argument>
 			<description>
+				Return an Image for a side of the CubeMap using the CubeMap.Side enum or an integer 0-5.
```
Comment (cbscribe) [inline doc/classes/CubeMap.xml]: - Return -> Returns
- Link to other classes: [Image]

Returns an [Image] of the CubeMap's given side. See [code]SIDE_*[/code] constants.

File: doc/classes/CubeMap.xml [hunk 4]
```diff
@@ -45,6 +49,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Return the width of the CubeMap.
```
Comment (cbscribe) [inline doc/classes/CubeMap.xml]: - Return -> Returns
- Per the writing guidelines, use possessive:

Returns the CubeMap's width.

File: doc/classes/CubeMap.xml [hunk 5]
```diff
@@ -71,6 +76,7 @@
 			<argument index="1" name="image" type="Image">
 			</argument>
 			<description>
+				Set an Image for a side of the CubeMap using the CubeMap.Side enum or an integer 0-5.
```
Comment (cbscribe) [inline doc/classes/CubeMap.xml]: - Set -> Sets
- see `get_side` comments above.

File: doc/classes/CubeMap.xml [hunk 6]
```diff
@@ -84,8 +90,10 @@
 	</methods>
 	<members>
 		<member name="lossy_storage_quality" type="float" setter="set_lossy_storage_quality" getter="get_lossy_storage_quality">
+			The lossy storage quality of the CubeMap if the storage mode is set to STORAGE_COMPRESS_LOSSY.
 		</member>
 		<member name="storage_mode" type="int" setter="set_storage" getter="get_storage" enum="CubeMap.Storage">
+			The storage mode of the CubeMap.
```
Comment (cbscribe) [inline doc/classes/CubeMap.xml]: - Per the writing guidelines, use possessive:

The CubeMap's storage mode. See [code]STORAGE_*[/code] constants.

File: doc/classes/CubeMap.xml [hunk 7]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CubeMap" inherits="Resource" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		A CubeMap is a 6 sided 3D texture.
 	</brief_description>
 	<description>
+		A CubeMap is a 6 sided 3D texture typically used for faking reflections. This usually delivers the best performace, it makes an object look as if it's reflecting it's surroundings.
```
Comment (cbscribe) [inline doc/classes/CubeMap.xml]: Second sentence is really two. Remove the comma and reword.

**Review by cbscribe — CHANGES_REQUESTED** [review]
A few changes to follow the writing guidelines: http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html

**Comment by NathanWarden** [comment]
@cbscribe Thanks for the comments, I adjust those and made corrections in get/set_side docs as they didn't correctly document the SIDE_* constants.

File: doc/classes/CubeMap.xml [hunk 8]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CubeMap" inherits="Resource" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		A CubeMap is a 6 sided 3D texture.
 	</brief_description>
 	<description>
+		A CubeMap is a 6 sided 3D texture typically used for faking reflections. It can be used to make an object look as if it's reflecting it's surroundings. This usually delivers much better performace than other reflection methods.
```
Comment (Noshyaar) [inline doc/classes/CubeMap.xml]: typos:
- performace -> performance
- reflecting it's surroundings -> reflecting its surroundings

Comment (NathanWarden) [reply doc/classes/CubeMap.xml]: Thanks, both fixed.

File: doc/classes/CubeMap.xml [hunk 9]
```diff
@@ -19,6 +21,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Returns the height of the CubeMap.
```
Comment (cbscribe) [inline doc/classes/CubeMap.xml]: - Per the writing guidelines, use possessive:

Returns the CubeMap's height.

File: doc/classes/CubeMap.xml [hunk 10]
```diff
@@ -45,6 +49,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Returns the width of the CubeMap.
```
Comment (cbscribe) [inline doc/classes/CubeMap.xml]: - Per the writing guidelines, use possessive:

Returns the CubeMap's width.

**Review by cbscribe — CHANGES_REQUESTED** [review]
Missed a couple of suggestions from the previous review.

File: doc/classes/CubeMap.xml (hunk 10 above)
Comment (NathanWarden) [reply doc/classes/CubeMap.xml]: Got those, thanks :)

**Review by cbscribe — APPROVED** [review]

**Comment by cbscribe** [comment]
Thanks for your contribution!

**Pull request merged**